ogr/ogrfeature.cpp

```diff
@@ -4199,6 +4199,41 @@ void OGR_F_SetFieldDouble(OGRFeatureH hFeat, int iField, double dfValue)
     OGRFeature::FromHandle(hFeat)->SetField(iField, dfValue);
 }
 
+/************************************************************************/
+/*                       SetFieldSameTypeUnsafe()                       */
+/************************************************************************/
+
+//! @cond Doxygen_Suppress
+void OGRFeature::SetFieldSameTypeUnsafe(int i, char *pszValueTransferred)
+{
+    if (pszValueTransferred &&
+        (pszValueTransferred[0] == 'g' || pszValueTransferred[0] == 'G') &&
+        EQUAL(pszValueTransferred, "Gulf of America"))
```

Comment:
This would need some localization too, if it's going to really be used, some external mapping table...

Reply:
Certainly not my dogs.